dotnet/deploy-packages-local.sh

```diff
@@ -0,0 +1,7 @@
+INSTALL_SHA="38787d2e9cadd408f541675e95fab7b9a687d559"
```

Comment:
I get how the SHA is set for the GitHub Action, but is this also supposed to work right with a local dev build or is that out of scope?

Reply:
That's out of scope. As said in the description, 2 things are not yet handled: use a local tracer, and modify where the files are modified to avoid having to revert modified files (as the build process do a lot of sed to change the version)